libs/feature-install-node-js/src/lib/install-node-js.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Apollo } from 'apollo-angular';
+import gql from 'graphql-tag';
+import { Observable } from 'rxjs';
+import { map, switchMap, tap } from 'rxjs/operators';
+
+import { InstallNodeJsStatus } from '../../../../server/src/graphql-types';
+
+@Component({
+  selector: 'angular-console-install-node-js',
+  templateUrl: './install-node-js.component.html',
+  styleUrls: ['./install-node-js.component.scss']
+})
+export class InstallNodeJsComponent {
+  installNodeJsStatus$?: Observable<boolean>;
+
+  constructor(
+    private readonly apollo: Apollo,
+    private readonly router: Router
+  ) {}
+
+  installNodeJs() {
+    this.installNodeJsStatus$ = this.apollo
+      .mutate({
+        mutation: gql`
+          mutation {
+            installNodeJs {
+              cancelled
+            }
+          }
+        `
+      })
+      .pipe(
+        map(
+          (response: { data: { installNodeJs: InstallNodeJsStatus } }) =>
+            response.data.installNodeJs
+        ),
+        switchMap(status => {
+          return this.apollo
+            .watchQuery({
+              pollInterval: 300,
+              query: gql`
+                query {
+                  installNodeJsStatus {
+                    downloadPercentage
+                    downloadSpeed
+                    success
+                    cancelled
+                    error
+                  }
+                }
+              `
+            })
+            .valueChanges.pipe(
+              map(
+                (result: {
+                  data: { installNodeJsStatus: InstallNodeJsStatus };
+                }) => result.data.installNodeJsStatus
+              ),
+              tap(res => console.log(res))
+            ) as Observable<InstallNodeJsStatus>;
+        }),
+        tap(({ success }: InstallNodeJsStatus) => {
+          if (success) {
```

Comment:
I think we should handle error as well.

You can still redirect to console but you can also use messanger service to show some error message.